src/sysnet/times.74

```diff
@@ -511,7 +569,7 @@ PDL:	BLOCK PDLLEN	;The Stack.
 	0
 
 DEBUG:	0		;-1 => Debugging.
-TCPP:	-1		;-1 => Try TCP time servers.
+TCPP:	0		;-1 => Try TCP time servers.
```

Comment:
This makes ITS try NCP (see ARTIM), which I don't think works. Either TCPP or CHAOSP should be -1, right?

Reply:
Yes. I didn't realize that using the JNAME of CTIMES automatically set CHAOSP, so leaving TCPP set to -1 is the right thing. I'll fix.